lib/formatters.rb

```diff
@@ -186,7 +186,22 @@ def header(data, fasta_mapper = nil)
       CSV.generate do |csv|
         first = data.first
         keys = fasta_mapper ? ['fasta_header'] : []
-        csv << (keys + first.keys).map(&:to_s) if first
+
+        firstKeys = first.keys
+
+        if firstKeys.include?("ec")
```

Comment:
I would deduplicate these 2 if's and wrap them in something like
```ruby
['ec', 'go'].each do |annotation|
   code
end
```

_[Original comment](https://github.ugent.be/unipept/unipept-cli/pull/55#discussion_r21549) by @bmesuere on Thu May 23 2019 at 10:36._